packages/mux-player/src/styles.css

```diff
@@ -112,7 +109,7 @@ media-theme {
   display: var(--media-controls, var(--media-mute-button));
 }
 ::part(captions menu-button) {
-  display: var(--media-controls, var(--media-captions-menu-button, contents));
```

Comment:
the default display value is no longer `contents`

Reply:
praise: glad you caught this